tools/editor/create_dialog.cpp

```diff
@@ -107,6 +108,9 @@ void CreateDialog::add_type(const String& p_type,HashMap<String,TreeItem*>& p_ty
 
 	}
 
+        const String& description = EditorHelp::get_doc_data()->class_list[p_type].brief_description;
```

Comment:
It's basically the same as making a new commit and rebase-squashing, but you can, instead of pushing a new commit, use ``git commit --amend`` to squash the currently staged changes directly in the previous commit.